packages/core/content-manager/admin/src/history/components/VersionInputRenderer.tsx

```diff
@@ -168,9 +168,9 @@ const CustomRelationInput = (props: RelationsFieldProps) => {
  * -----------------------------------------------------------------------------------------------*/
 
 const CustomMediaInput = (props: VersionInputRendererProps) => {
-  const {
-    value: { results, meta },
-  } = useField(props.name);
+  const { value } = useField(props.name);
+  const results = value ? value.results : [];
+  const meta = value ? value.meta : { missingCount: 0 };
```

Comment:
nice, should we maybe do the same for CustomRelationInput to be safe?

Reply:
yeah, I tried to replicate the error for the relations and there is not error but for sure we can add a preventive check